### DISCUSSION TITLE
Delete a feed

### DISCUSSION THREAD

**Discussion by fuermosi777** [mxpv/podsync#736] [General]
Sometimes I want to delete a feed completely but if I just remove `[feed.ID]` section from the config file, I can still see the folder and the xml being served. I tried to look at the `/data` folder to do a manual deletion, but it's empty for some reasons (I am using docker to set up the Podsync).

Could you help advise how to properly remove a feed and its downloaded files? Thank you.

**Comment by AdamCurtisVT** [comment]
Are you restarting the container after modifying the config file? Podsync reads the config file when it starts so any changes made afterwards won't have an impact.

**Comment by beejsbj** [comment]
> Are you restarting the container after modifying the config file? Podsync reads the config file when it starts so any changes made afterwards won't have an impact.

How do you restart the container?
